pkg/datasource/formatters/json/json.go

```diff
@@ -115,6 +116,12 @@ func (f *Formatter) addSubFields(accessors []datasource.FieldAccessor, prefix st
 	}
 
 	ctr := -1
+
+	// sort alphabetically
+	sort.Slice(accessors, func(i, j int) bool {
+		return accessors[i].Name() < accessors[j].Name()
+	})
```

Comment:
I like this change, but I also like the idea of having things in this order:

1. Runtime-specific stuff
2. Kubernetes-specific stuff
3. Gadget-specific stuff

Where 1 and 2 can be exchanged, it's also fine.

I don't know, what do others think?

Reply:
Main reason was: the Go JSON marshaler sorts by keys as well. As a human I also find it easier to navigate :)

Reply:
Easier to navigate but, IMO, not very good looking. Anyway, given that no one else commented, let's go ahead. Once it's merged and people will start using it, we will realize if people like it or not.